### PR TITLE
Updated the sections Accessing the SSUI and Navigating the SSUI with …

### PR DESCRIPTION
**Pull request by suyogsainkar**
…the changes for 4.6 GA.

BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1545504

@cbudz Hi Chris, 
Would you mind peer reviewing the updates in this PR? Let me know what you think. 
Thanks, 
Suyog

### CODE REVIEW

File: doc-Self_Service_User_Interface_Guide/topics/Navigating_Self_Service_UI.adoc [hunk 1]
```diff
@@ -28,7 +28,12 @@ Clicking the menu icon (image:ssui-administration-ui-icon.png[Administration UI
 
 Help::
 
-Click the help icon to access the options to open *Documentation*, *Red Hat Customer Portal*, and *About*. Click *About* to see information about your Self Service environment, such as the logged in user name and role, the browser details, and more importantly, the version number of your Self Service user interface, which can be useful for support purposes.
+Click the help icon to access the options to open *Documentation*, *Red Hat Customer Portal*, and *About*. Click *About* to see information about your Self Service environment, such as the logged in user name and role, the browser details, and importantly, the Self Service user interface version number, which can be useful for support purposes.
+
+[NOTE]
```
Comment (cbudz) [inline doc-Self_Service_User_Interface_Guide/topics/Navigating_Self_Service_UI.adoc]: Hey Suyog. I would recommend breaking this NOTE into two sentences. System administrators can change....

Comment (suyogsainkar) [reply doc-Self_Service_User_Interface_Guide/topics/Navigating_Self_Service_UI.adoc]: Updated, thanks.

**Comment by suyogsainkar** [comment]
@cbudz Thanks Chris, I've applied your suggestion in the latest commit.

**Comment by cbudz** [comment]
Hey @suyogsainkar 

Changes look good, merging this request.

-Chris

**Comment by suyogsainkar** [comment]
@cbudz thanks for reviewing. And no problem - I've resolved the conflict so merging now.

**Pull request merged**